project-base/app/config/shopsys-routing/routing_front_cs.yaml

```diff
@@ -41,3 +41,6 @@ front_stores:
 
 front_complaint_detail:
     path: /zakaznik/detail-reklamace
+
+front_social_network_login_page:
+    path: /social-login
```

Comment:
It looks like a forgotten translation. Why isn't it translated?

Reply:
for some reason, all the SF routes need to be put in the routes, even though they are not translated - https://github.com/shopsys/shopsys/blob/17.0/project-base/storefront/config/routes.ts. This route does not need to be translated but can be :shrug: I hope the duplication of the definitions will be solved in SSP-3404 after the app router is finished